build.gradle

```diff
@@ -97,13 +95,10 @@ task diff << {
 					standardOutput = os
 				}
 
-				def output = os.toString()
+				def errorOutput = os.toString()
```

Comment:
Since we don't have the `println "${replicationFiles}"` we don't know which directory the error was in, also since we don't have `println replicationFile.name` we dont' know which bundle the error happened in.  We need to add those back somehow.